src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -53,11 +53,13 @@ New Relic has several Helm charts for the different components which offer diffe
 * [`nri-metadata-injection`](https://github.com/newrelic/k8s-metadata-injection/tree/main/charts/nri-metadata-injection): Sets up a minimal `MutatingAdmissionWebhook` that injects a couple of environment variables in the containers. These contain metadata about the cluster and New Relic installation and will be later picked up by applications instrumented using APM, allowing to [correlate APM and infrastructure data](/docs/integrations/kubernetes-integration/link-your-applications/link-your-applications-kubernetes/).
 * [`nri-statsd`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-statsd): New Relic StatsD integration.
 
-For convenience, New Relic provides the [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart, which acts as a wrapper or a meta-package for the individual charts mentioned above. While it's possible to install them separately, we strongly recommend doing so using the [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) instead, because:
+Although you can install these components separately, we strongly recommend you use the [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart. New Relic provides this chart, which acts as a wrapper or a meta-package for the individual charts mentioned above. The use of this chart enables you these advantages: 
+
 - It provides full control over which components are installed.
 - It ensures that their installed versions are compatible with each other.
 - It ensures that their configuration values are consistent across the installed charts.
-- You can still configure each component individually as a [Helm dependency](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#overriding-values-from-a-parent-chart), using the configuration parameters mentioned [here](#configure).
+
+If you decide to install the components separately, you can configure each component individually as a [Helm dependency](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#overriding-values-from-a-parent-chart), using the configuration parameters mentioned [here](#configure).
```

Comment:
Sorry, this probably was badly expressed. This only applies when installing the components together. A clearer way to say this could be:

```
- Each component is installed as a separate [Helm dependency](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#overriding-values-from-a-parent-chart) that you can configure using the parameters mentioned [here](#configure)
```